Dockerfile

```diff
@@ -1,18 +1,22 @@
-ARG VERSION=1.15.7-buster
-FROM golang:$VERSION
+FROM centos:8
```

Comment:
a comment here with the reasoning would be great

Reply:
totally agree, multitasking mode is killing me today :(